todo_app/app.py

```diff
@@ -1,22 +1,207 @@
 import os.path
-from flask import Flask,escape,request,Response,render_template, redirect
+import os
+import requests
+from flask import Flask,escape,request,Response,render_template, redirect,url_for
 import todo_app.data.session_items as session
 
 app = Flask(__name__)
-app.secret_key = "super secret key"
+key = os.getenv("TRELLO_KEY")
+token = os.getenv("TRELLO_TOKEN")
```

Comment:
You should update .env.template to have placeholder values for these and update the README so that the reader knows how to get the app running